multisig/proposals/ethereum/vip-035/index.ts

```diff
@@ -0,0 +1,78 @@
+import { parseUnits } from "ethers/lib/utils";
+
+import { NETWORK_ADDRESSES } from "../../../../src/networkAddresses";
+import { makeProposal } from "../../../../src/utils";
+
+const { ethereum } = NETWORK_ADDRESSES;
+
+const UPPER_BOUND_RATIO = parseUnits("1.01", 18);
+const LOWER_BOUND_RATIO = parseUnits("0.99", 18);
+const rsETH_ONE_JUMP_REDSTONE_ORACLE = "0xf689AD140BDb9425fB83ba6f55866447244b5a23";
+const rsETH_ONE_JUMP_CHAINLINK_ORACLE = "0xD63011ddAc93a6f8348bf7E6Aeb3E30Ad7B46Df8";
+export const rsETH = "0xA1290d69c65A6Fe4DF752f95823fae25cB99e5A7";
+export const vrsETH = "0xDB6C345f864883a8F4cae87852Ac342589E76D1B";
+const BOUND_VALIDATOR = "0x1Cd5f336A1d28Dff445619CC63d3A0329B4d8a58";
+export const INITIAL_SUPPLY = parseUnits("2", 18); // TBD
+export const SUPPLY_CAP = parseUnits("8000", 18);
+export const BORROW_CAP = parseUnits("3600", 18);
+const CHAINLINK_rsETH_FEED = "0x03c68933f7a3F76875C0bc670a58e69294cDFD01";
+const REDSTONE_rsETH_FEED = "0xA736eAe8805dDeFFba40cAB8c99bCB309dEaBd9B";
+const STALE_PERIOD_26H = 60 * 60 * 26; // 26 hours (pricefeeds with heartbeat of 24 hr) ;
+const CF = parseUnits("0.8", 18);
+const LT = parseUnits("0.85", 18);
+
+export const vip035 = () => {
+  return makeProposal([
+    // Configure Oracle
+    {
+      target: BOUND_VALIDATOR,
+      signature: "setValidateConfig((address,uint256,uint256))",
+      params: [[rsETH, UPPER_BOUND_RATIO, LOWER_BOUND_RATIO]],
+    },
+    {
+      target: ethereum.REDSTONE_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[rsETH, CHAINLINK_rsETH_FEED, STALE_PERIOD_26H]],
+    },
+    {
+      target: ethereum.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[rsETH, REDSTONE_rsETH_FEED, STALE_PERIOD_26H]],
+    },
```

Comment:
The other way...
```suggestion
    {
      target: ethereum.REDSTONE_ORACLE,
      signature: "setTokenConfig((address,address,uint256))",
      params: [[rsETH, REDSTONE_rsETH_FEED, STALE_PERIOD_26H]],
    },
    {
      target: ethereum.CHAINLINK_ORACLE,
      signature: "setTokenConfig((address,address,uint256))",
      params: [[rsETH, CHAINLINK_rsETH_FEED, STALE_PERIOD_26H]],
    },
```

Reply:
Fixed [05923b8](https://github.com/VenusProtocol/vips/pull/304/commits/05923b879a879ecd7b257ccae5de68e766188b95)